### ISSUE TITLE
[Page Persistance

### ISSUE THREAD

**Issue by woorst** [michael-lazar/rtv#259]
It would be great if when you returned to a page that you have previously visited in the current rtv session it is in that same state you left it in.
For example: you open a submission, scroll down a dozen comments then exit the submission. When you reopen that same submission you should be at that same comment you scrolled to.

**Comment by michael-lazar** [comment]
Agreed, this would be a neat feature. We would need to somehow cache the **SubmissionContent()** and the **objects.Navigator()** state, but beyond that I don't see anything that would make this impossible.

**Comment by vdmkenny** [comment]
I would enjoy this feature. I find myself having to close rtv often and reopen it later.
